tensorflow_addons/rnn/peephole_lstm_cell.py

```diff
@@ -39,14 +39,15 @@ class PeepholeLSTMCell(tf.keras.layers.LSTMCell):
 
     Example:
 
-    ```python
-    # Create 2 PeepholeLSTMCells
-    peephole_lstm_cells = [PeepholeLSTMCell(size) for size in [128, 256]]
-    # Create a layer composed sequentially of the peephole LSTM cells.
-    layer = RNN(peephole_lstm_cells)
-    input = keras.Input((timesteps, input_dim))
-    output = layer(input)
-    ```
+    >>> # Create 2 PeepholeLSTMCells
+    >>> peephole_lstm_cells = [PeepholeLSTMCell(size) for size in [128, 256]]
+    >>> # Create a layer composed sequentially of the peephole LSTM cells.
+    >>> layer = tf.keras.layers.RNN(peephole_lstm_cells)
+    >>> input = tf.keras.Input((100, 100))
+    >>> output = tf.keras.layers.Layer(input)
+
+    <... Note: Values used for timesteps and input_dim have been used purely
```

Comment:
Is this warning? I think we can at least print output.

Reply:
I'll include the output of `output` but it's only going to be displayed as a `Layer` object. I think the example tries to depict the way in which the `PeepholeLSTMCells` can be utilized, it doesn't necessarily demonstrate the full process. That's why I included a short note stating that the values used for `input` have no significance.

Reply:
Oh I see. Can we change the functional API to
```python
>>> output, state = rnn(input)
>>> output
>>> state
```
This is much more informative.

Reply:
@WindQAQ Sorry for the late reply I somehow missed this comment. When I try
```python
>>> keras.layers.RNN(peephole_lstm_cells,return_state=True)
>>> layer.states
```
it displays `([None, None], [None, None])`. I'm a little confused on how to view the state of the layer.